software/tests/test_graphs.py

```diff
@@ -1,30 +1,29 @@
 #!/usr/bin/env python3
 # -*- coding: UTF-8 -*-
+
+import os
 import sys
+
 if not (sys.version_info.major == 3 and sys.version_info.minor > 5):
     print("Python version %s.%s not supported version 3.6 or above required - exiting" % (sys.version_info.major,sys.version_info.minor))
     sys.exit(1)
 
-import os
+
 for path in [os.getcwd(),"software/Util","software/SchemaTerms","software/SchemaExamples"]:
-  sys.path.insert( 1, path ) #Pickup libs from local  directories
+  sys.path.insert(1, path) #Pickup libs from local directories
 
 import unittest
 import os
```

Comment:
do we need this when it's already on line 4?

Reply:
This was a move?